quipucords/quipucords/settings.py

```diff
@@ -430,14 +430,14 @@ def app_secret_key_and_path():
 }
 
 
-def warn_on_deprecation(message, category, filename, lineno, file=None, line=None):  # noqa: PLR0913
-    """Redirect deprecation warnings to our logger."""
-    logger = logging.getLogger("quipucords.warnings")
-    logger.warning(f"{message} (from {filename}:{lineno})")
+# def warn_on_deprecation(message, category, filename, lineno, file=None, line=None):  # noqa: PLR0913
+#     """Redirect deprecation warnings to our logger."""
+#     logger = logging.getLogger("quipucords.warnings")
+#     logger.warning(f"{message} (from {filename}:{lineno})")
 
 
-warnings.showwarning = warn_on_deprecation
-warnings.simplefilter("always", DeprecationWarning)
+# warnings.showwarning = warn_on_deprecation
+# warnings.simplefilter("always", DeprecationWarning)
```

Comment:
I commented these out only because **somehow** this was causing a recursion error when importing quipucords in a jupyter notebook